tdrs-backend/tdpservice/stts/serializers.py

```diff
@@ -23,6 +23,21 @@ def get_code(self, obj):
         return obj.code
 
 
+class STTUpdateSerializer(serializers.ModelSerializer):
+    """STT serializer."""
+
+    class Meta:
+        """Metadata."""
+
+        model = STT
+        fields = ["id"]
+        extra_kwargs = {"id": {"read_only": False}}
```

Comment:
Can we say a little more about what setting `"read_only": False` is doing on the ID field? 

Would this allow the ID of an STT to be changed? Do we want the ID of an STT to be changeable?

Reply:
This allows us to update the user's STT by passing the ID, otherwise the `set_profile` endpoint would error. Django won't allow you to update a PK like that. If you did try, Django would create a new object instead of updating the old one. Obviously, we wouldn't try to do this.

Reply:
I could add a unique constraint to the STT name to prevent that as well, but we would need to verify there would never be two tribes or territories with the same name. It would also require writing some tests around it. If you think we should do that I think we should create a separate task for it. Thoughts?

Reply:
Gotcha! In that case I don't think any code changes are needed here, thanks for the explanation!